R/common.R

```diff
@@ -58,7 +58,7 @@ WORKER_CONFIG_DEFAULT <- "localhost"
 ## the data layout. When either a worker or controller connects to the
 ## db they'll compare their version against this and report back if
 ## they disagree.
-rrq_schema_version <- "0.4.0"
+rrq_schema_version <- "0.4.1"
```

Comment:
```suggestion
rrq_schema_version <- "0.8.0"
```

as in, the version that will be the schema from 0.8.0